onboardingtaskapp.client/src/components/Sale/Create.jsx

```diff
@@ -68,10 +99,25 @@ export default function Create() {
             if (!response.ok) {
                 throw new Error('Network response was not ok');
             }
-            // Handle success
-            window.location.replace("/sales");
+            // Handle success           
             console.log('Store created successfully');
```

Comment:
1. Please check my previous comment about console.logs
2. Please implement the API endpoint that I mentioned above